src/mount/sauna_client.cc

```diff
@@ -3496,7 +3496,8 @@ void fs_init(FsInitParams &params) {
 			params.prefetch_xor_stripes,
 			std::max(params.bandwidth_overuse, 1.));
 	write_data_init(params.write_cache_size, params.io_retries, params.write_workers,
-			params.write_window_size, params.chunkserver_write_timeout_ms, params.cache_per_inode_percentage);
+			params.write_window_size, params.chunkserver_write_timeout_ms, 
```

Comment:
Please, remove the trailing space at the end of this line, i.e., extra whitespace after `params.chunkserver_write_timeout_ms, `

Reply:
Please, fix also the typo in the commit message: `mound` instead of `mount`.

Reply:
Done